app/webroot/css/user/settings.css

```diff
@@ -43,4 +33,27 @@
 .options .experimental-info {
     margin: 5px;
     color: #EE0000;
+}
+
+.settings-form{
+    padding: 20px;
+    margin: 20px auto;
+}
+
+.settings-form h2{
+    margin: 0 0 40px 0;
+    text-align: center;
```

Comment:
I think it would be better to not center the text here. For consistency with the rest of the pages.
On every div block that had a class "module", keep the title aligned left.

The title on the "login" and "password forgotten" pages can remain centered because the blocks were not "modules".